dashboard/lib/build_dashboard_page.dart

```diff
@@ -107,56 +109,18 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
         child: Material(
           color: Colors.transparent,
           child: FocusTraversalGroup(
-            child: Column(
-              mainAxisSize: MainAxisSize.min,
+            child: ListView(
               children: <Widget>[
-                DropdownButton<String>(
-                  value: _buildState.currentRepo,
-                  icon: const Icon(
-                    Icons.arrow_downward,
-                  ),
-                  iconSize: 24,
-                  elevation: 16,
-                  underline: Container(
-                    height: 2,
-                  ),
-                  onChanged: (String? selectedRepo) {
-                    repo = selectedRepo;
-                    _updateNavigation(context);
-                  },
-                  items: _buildState.repos.map<DropdownMenuItem<String>>((String value) {
-                    return DropdownMenuItem<String>(
-                      value: value,
-                      child: Text(value, style: theme.primaryTextTheme.bodyText1),
-                    );
-                  }).toList(),
-                ),
-                DropdownButton<String>(
-                  value: _buildState.currentBranch,
-                  icon: const Icon(
-                    Icons.arrow_downward,
-                  ),
-                  iconSize: 24,
-                  elevation: 16,
-                  underline: Container(
-                    height: 2,
-                  ),
-                  onChanged: (String? selectedBranch) {
-                    branch = selectedBranch;
-                    _updateNavigation(context);
-                  },
-                  items: _buildState.branches.map<DropdownMenuItem<String>>((String value) {
-                    return DropdownMenuItem<String>(
-                      value: value,
-                      child: Text(value, style: theme.primaryTextTheme.bodyText1),
-                    );
-                  }).toList(),
-                ),
+                if (_smallScreen) ..._slugSelection(context, _buildState),
                 TextButton(
                   child: const Text('Vacuum GitHub Commits'),
                   onPressed: _buildState.refreshGitHubCommits,
                 ),
-                FilterPropertySheet(_filter),
+                Row(
+                  children: [
+                    Expanded(child: Center(child: FilterPropertySheet(_filter))),
```

Comment:
Is this Expanded the reason that the filter dialog now covers the entire page? This means that you can't mouse around or click on items (particular row/column headers) with the dialog open.

Reply:
yeah sorry about that, was a side effect of me trying to center the dialog but now it covers the entire screen. Was wondering if there is a good way to center it

Reply:
Isn't Center enough? Its child is a Center.

Reply:
Yeah I thought that way previously but Center itself does not quite work, not sure why

Reply:
Are you looking into this or should I file an issue so we don't forget? (Would that be a flutter/flutter issue?)

Reply:
Hmmm, it would be nice if it was repositionable.

Reply:
yes sir feel free to file an issue so that we can address it later. Oh I am not looking at it now. I am currently doing a release. The last time I looked at it I wasn't very successful in getting it reposition-able. Might need some help from my mentor @CaseyHillers

Reply:
Hello flar, I am back from release. I have updated a pr at https://github.com/flutter/cocoon/pull/1667 to address your suggestions.